src/os/bluestore/bluestore_tool.cc

```diff
@@ -525,6 +553,8 @@ int main(int argc, char **argv)
     }
     fs->umount();
     delete fs;
+  } else if (action == "log-dump") {
```

Comment:
bluefs-log-dump? since other bluefs stuff is prefixed.

Reply:
ok